docs/Documentation/run_RBM.md

```diff
@@ -89,9 +87,9 @@ This will create the executable,
 
 For this example, copy **rout** to the folder **Test_Case**
 
-Create control file, **salmon.inp_DA**, to run the modified Lohmann routing model using the file, **Rout.Cells.init**, the first time and the file, **Rout.Cells**, if the routing model, **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold parameters. The routing model requires a file with unit hydrographs for each cell, or, in this case, a file, **UH_ALL**, with a single unit hydrograph that is applied to all grid cells.
+Create the control file **salmon.inp_DA** (see [Figure 3](../figures/tutorial_f3.png) for an example) to run the modified Lohmann routing model using the file **Rout.Cells.init** (for the first time) or the file **Rout.Cells** if the routing model **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold parameters. The routing model requires a file with unit hydrographs for each cell, or, in this case, a file, **UH_ALL**, with a single unit hydrograph that is applied to all grid cells.
```

Comment:
- Split the first sentence into two - "User needs to prepare a control file for running the modified Lohmann routing model (salmon.inp_DA; also see Figure 3 for an example). The `Name of station file` specified in this control file should be set to the file `Rout.Cells.init` if this is the first time running the routing model, and to the file `Rout.Cells` if the routing model has already been run before using the same routing parameters."
- The last sentence - I'd suggest you delete this sentence; instead, point to the Lohmann routing input documentation page and say something like "most parameters in this control file are the same as the original Lohmann routing model, and see here (link) for detailed instructions". If any parameter is different than the original Lohmann routing, describe it. (You described the UH box input file here; but there are a couple of other important input parameters/files as well)